src/answers/services/s3Service.ts

```diff
@@ -35,10 +40,29 @@ export class S3Service {
       Bucket: this.S3_BUCKET,
       Key: `${this.S3_FOLDER}/${uuid}.${ext}`,
       Body: buffer,
-      ACL: 'public-read',
+      ACL: isPublic ? 'public-read' : 'private',
```

Comment:
This will break exports of private images as we put in links. If the files are no longer public-read then they cannot be read by anybody apart from the bucket owner (aws account owner) or with a link signed by the owner (which will be ephemeral).

Reply:
Also, if we are always signing urls for viewing then this property means nothing because the user is always viewing it as the aws account owner due to the presigned url